C/include/c4Listener.h

```diff
@@ -58,6 +79,9 @@ extern "C" {
         C4ListenerAPIs apis;            ///< Which API(s) to enable
         C4TLSConfig* tlsConfig;         ///< TLS configuration, or NULL for no TLS
 
+        C4ListenerHTTPAuthCallback httpAuthCallback; ///< Callback for HTTP auth
+        void* callbackContext;
```

Comment:
Ambiguous name here, could it be renamed?

Reply:
It's a context for all the callbacks in that struct. There's currently just one, but we could add more. (See C4ReplicatorConfig, which has a field with the same name.)

The one in C4TLSConfig just has a different name to avoid confusion.